tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -4809,4 +4808,24 @@ public function userGetsTheDriveItemInSpace(string $user, string $file, string $
 			)
 		);
 	}
+
+	/**
+	 * @Given /^user "([^"]*)" has expired the membership of user "([^"]*)" from space "([^"]*)"$/
+	 *
+	 * @param  string $user
+	 * @param  string $memberUser
+	 * @param  string $spaceName
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userHasExpiredTheMembershipOfUserFromSpace(
+		string $user,
+		string $memberUser,
+		string $spaceName
+	): void {
+		$rows['expireDate'] = $this->featureContext->getExpiryDateTime()->format('Y-m-d\\TH:i:sP');
```

Comment:
then you can also use it here
```suggestion
		$rows['expireDate'] = $this->featureContext->formatExpiryDateTime('Y-m-d\\TH:i:sP');
```